src/main/webapp/index.xhtml

```diff
@@ -52,7 +52,14 @@
                     #{item.name}
                 </h:column>
                 <h:column>
-                    <i class="far fa-trash-alt"></i>
+                	<h:form>
+                		<h:commandButton
+                			id="deleteItem"
+                			valuse="delete"
+                			icon = "fa-trash-alt"
+                			action="#{localTradeList.deleteItem(item)}">
+                    	</h:commandButton>
+                    </h:form>

```

Comment:
- Doesn't need another form; the previous form can be extended.
- Page reloads when delete is clicked.
- "valuse" typo

Reply:
I think I should have fixed these